Add a pg_config path for bundler

komainu8 opened the pull request.

komainu8 committed:
Add a pg_config path for bundler

kou commented on ansible/almalinux-8-primary/playbook.yml:
```diff
@@ -173,6 +173,11 @@
+      become_user: vagrant
+      shell: |
+        bundle config build.pg \
+          --with-pg-config=/usr/pgsql-{{ postgresql_version.series }}/bin/pg_config
```
Could you also add `ansible_ssh_pipelining: true` like other `shell` tasks?
I don't remember the detail but `shell` and `become_user` may fail without it.

komainu8 committed:
Add a missing required vars

Because "shell" and "become_user" may fail without "ansible_ssh_pipelining: true".
